src/utils.js

```diff
@@ -112,7 +113,10 @@ module.exports.readConfigFile = () => {
     // these locations must be under the user's project folder
     app_location: path.join(process.cwd(), app_location),
     api_location: path.join(process.cwd(), api_location),
-    app_artifact_location: path.join(process.cwd(), app_location, app_artifact_location),
+    app_artifact_location:
+      detectRuntime(path.join(process.cwd(), app_location)) === RuntimeType.node
+        ? path.join(process.cwd(), app_location, app_artifact_location)
+        : path.join(process.cwd(), app_location, "bin", "Debug", "netstandard2.1", "publish", app_artifact_location),
```

Comment:
We can maybe extract this code into a separate function. I can see this logic get complicated once we will have to support more technologies (python, java...).

Reply:
Not sure if that'll be the case, remember that this is the front-end runtime, so I don't know what other front-ends we'd support. dotnet is a bit of a special case since there's a WASM story, but we'd have to find out if there were any other supported WASM front-ends (Java to WASM, Python to WASM, etc.) that oryx will understand.

Reply:
My bad 🤦